procrastinate/migration.py

```diff
@@ -5,6 +5,8 @@
 
 @utils.add_sync_api
 class Migrator:
+    version = "0.2.1"
```

Comment:
Will we need to ensure this version number is in sync with that in the SQL file?

Reply:
We do. Fortunately, the test `tests/integration/test_healthchecks.py::test_healthchecks` both test that the healthcheck work as expected with a working database, AND that running healthcheck on a new database works, including this version number comparison. So it tests that the version numbers are synced. 😄 

I do agree it's not crystal clear. I don't have a better idea atm, however. :/

Reply:
The alternative is to parameterize the migration, which we'll probably have to do anyway at some point, but if we can avoid that for a while, it might make things simpler.

A third option would be to remove the `INSERT INTO procrastinate_versions` query from the migration and manage it with a special system. But I'd rather have the whole migrations be "consistent". If there's a way we could have the migration in 1 place and that place be SQL code, I'd love that.

Reply:
Maybe we could have the insert into the queries.sql, and let the Migrator run that query after doing its migrations?